profiles/use.desc

```diff
@@ -97,6 +97,7 @@ freetds - Add support for the TDS protocol to connect to MSSQL/Sybase databases
 ftp - Add FTP (File Transfer Protocol) support
 gd - Add support for media-libs/gd (to generate graphics on the fly)
 gdbm - Add support for sys-libs/gdbm (GNU database libraries)
+generic-uki - Install a generic initramfs and Unified Kernel Image (EXPERIMENTAL)
```

Comment:
```suggestion
generic-uki - Install a generic initramfs and Unified Kernel Image
```

I think it's time to drop the `(EXPERIMENTAL)` since we've just expanded this to vanilla-kernel and are preparing to drop the guard variables from the eclass.